app/kuma-dp/pkg/dataplane/envoy/remote_bootstrap.go

```diff
@@ -130,6 +186,16 @@ func (b *remoteBootstrap) requestForBootstrap(ctx context.Context, url *net_url.
 	if cfg.DataplaneRuntime.Token != "" {
 		token = cfg.DataplaneRuntime.Token
 	}
+
+	resourceMeta, err := b.resourceMetadata(cfg.DataplaneRuntime.Resources)
+	if err != nil {
+		return nil, err
+	}
+
+	for k, v := range resourceMeta {
+		params.DynamicMetadata[k] = v
```

Comment:
I think it would be more clear to define this in BootstrapRequest explicitly

Reply:
You mean not in metadata but as a new field there?

Reply:
yes